index.html

```diff
@@ -1,267 +1,256 @@
       </div>
-
-      <div class="parent--container">
-        <div class="first--child">
-          <a href="#"><img src="images/4th mobile pic.svg" alt="" /></a>
-        </div>
-
-        <div class="second--container">
-          <h2>Multi-Post Stories</h2>
-          <ul class="ul--list">
-            <li>CANOPY</li>
-            <li>Back End Dev</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="skill--list">
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>Javascript</li>
-          </ul>
-          <button type="button" class="see--project">
-            See project
-          </button>
-        </div>
+      <div class="second--child order-two">
+        <h2>Multi-Post Stories</h2>
+        <ul class="ul--list">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="skill--list">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>Javascript</li>
+        </ul>
+        <button type="button" class="see--project">
+          See project
+        </button>
       </div>
-    </section>
-
-    <section class="about--section">
-      <div>
-        <h2>About Myself</h2>
-        <p class="about--text">
-          Hello I'm a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don't hesitate to
-          contact me.
+    </div>
+    <div class="parent--container">
+      <div class="first--child">
+        <a href="#"><img src="images/3rd pic.svg" class="img-three" alt="" /></a>
+      </div>
+      <div class="second--child">
+        <h2>Tonic</h2>
+        <ul class="ul--list">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
         </p>
-        <p class="hero-connect-text">LET'S CONNECT</p>
-        <ul class="hero-icon-container">
-          <li>
-            <a href="https://twitter.com/AyokunnumiA">
-              <img src="images/Vector.svg" alt="" />
-            </a>
-          </li>
-          <li>
-            <a href="https://www.linkedin.com/in/ayokunnumiomololu">
-              <img src="images/Linkedin icon.svg" alt="" />
-            </a>
-          </li>
+        <ul class="skill--list">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>Javascript</li>
+        </ul>
+        <button type="button" class="see--project">
+          See project
+        </button>
+      </div>
+    </div>
```

Comment:
[OPTIONAL]
- [ ] Kindly stick to the `Figma` design specs as much as possible, and try to use the same `paddings` and `margins` as the `Figma` to reach the best look for your project, the same goes for every card on your `Works` section.

| Your design | Figma Design |
| ------------- | ------------- |
| ![Screenshot 2023-06-11 001250](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/102612396/571c7319-00c1-4343-b3a3-c30e9fe73439) | ![image](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/102612396/926963cf-5423-432e-99c9-7db87d4f7a35) |